app/views/site/_search_options.html.erb

```diff
@@ -258,21 +253,22 @@
           </div>
           <div class="row">
             <div class="col-sm-12">
-              <div id="selection_div" class="collapse"></div>
+              <div id="create-annotation-panel" class="collapse"></div>
             </div>
           </div>
         </div>
       </div>
       <script type="text/javascript" nonce="<%= content_security_policy_script_nonce %>">
         <% if user_signed_in? %>
 
-          var targetPlotString = '<%= params[:gene_set].nil? && params[:gene].nil? ? 'cluster-plot' : 'scatter-plot' %>';
+          // TODO (SCP-2962): Support "Create Annotations" for spatial scatter plots
+          var targetPlotId = 'scatter-plot-0';
```

Comment:
I believe this can be replaced with `window.SCP.getScatterPlots()[0].plotId`, which is nice to avoid hardcoding ID formats, (even if it requires a double-check that window.SCP.getScatterPlots() has length)

Reply:
Done in latest code.